test/requirements.txt

```diff
@@ -0,0 +1,3 @@
+-r ../requirements.txt
+mock==2.0.0
+nose==1.3.7
```

Comment:
actually i would prefer `pytest` since i already use it in the travis configuration :-)

Reply:
Fine by me :) Fixed this in be5d460cf48a4457ae11b5629ba0c83b6d6b602a